src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dto/validator/NationalityValidator.java

```diff
@@ -0,0 +1,260 @@
+        modifiableListOfNationalities.add("Serbian");
+        modifiableListOfNationalities.add("Sierra Leonean");
+        modifiableListOfNationalities.add("Singaporean");
+        modifiableListOfNationalities.add("Slovak");
+        modifiableListOfNationalities.add("Slovenian");
+        modifiableListOfNationalities.add("Solomon Islander");
+        modifiableListOfNationalities.add("Somali");
+        modifiableListOfNationalities.add("South African");
+        modifiableListOfNationalities.add("South Korean");
+        modifiableListOfNationalities.add("South Sudanese");
+        modifiableListOfNationalities.add("Spanish");
+        modifiableListOfNationalities.add("Sri Lankan");
+        modifiableListOfNationalities.add("St Helenian");
+        modifiableListOfNationalities.add("St Lucian");
+        modifiableListOfNationalities.add("Stateless");
+        modifiableListOfNationalities.add("Sudanese");
+        modifiableListOfNationalities.add("Surinamese");
+        modifiableListOfNationalities.add("Swazi");
+        modifiableListOfNationalities.add("Swedish");
+        modifiableListOfNationalities.add("Swiss");
+        modifiableListOfNationalities.add("Syrian");
+        modifiableListOfNationalities.add("Taiwanese");
+        modifiableListOfNationalities.add("Tajik");
+        modifiableListOfNationalities.add("Tanzanian");
+        modifiableListOfNationalities.add("Thai");
+        modifiableListOfNationalities.add("Togolese");
+        modifiableListOfNationalities.add("Tongan");
+        modifiableListOfNationalities.add("Trinidadian");
+        modifiableListOfNationalities.add("Tristanian");
+        modifiableListOfNationalities.add("Tunisian");
+        modifiableListOfNationalities.add("Turkish");
+        modifiableListOfNationalities.add("Turkmen");
+        modifiableListOfNationalities.add("Turks and Caicos Islander");
+        modifiableListOfNationalities.add("Tuvaluan");
+        modifiableListOfNationalities.add("Ugandan");
+        modifiableListOfNationalities.add("Ukrainian");
+        modifiableListOfNationalities.add("Uruguayan");
+        modifiableListOfNationalities.add("Uzbek");
+        modifiableListOfNationalities.add("Vatican citizen");
+        modifiableListOfNationalities.add("Venezuelan");
+        modifiableListOfNationalities.add("Vietnamese");
+        modifiableListOfNationalities.add("Vincentian");
+        modifiableListOfNationalities.add("Wallisian");
+        modifiableListOfNationalities.add("Welsh");
+        modifiableListOfNationalities.add("Yemeni");
+        modifiableListOfNationalities.add("Zambian");
+        modifiableListOfNationalities.add("Zimbabwean");
+        nationalities = Collections.unmodifiableList(modifiableListOfNationalities);
+    }
+
+    @Override
+    public boolean isValid(Object object, ConstraintValidatorContext constraintValidatorContext) {
+        if (object instanceof GeneralPartnerDataDto) {
```

Comment:
you can use pattern matching here so you don't have to cast the object, use:
    `if (object instanceof GeneralPartnerDataDto dto)`
then you don't need
`GeneralPartnerDataDto dto = (GeneralPartnerDataDto)object;`

Reply:
Updated